markup_fmt/tests/fmt/jinja/attributes/tag-or-block-in-attr-value.jinja

```diff
@@ -0,0 +1,11 @@
+<form class="form-control" action='{% url "my_app:subscribe" %}' method="post">
+  {{ whatever }}
+</form>
+<a
+    href='https://{{ request.get_host }}{% url "newsletter_verify" uuid=subscription.verification_token %}'
+>Subscribe to the newsletter!</a>
+<div
+  class="first-class {% block extra_classes %}{% endblock %} modal fade">
```

Comment:
Is this `>` added unexpectedly?

```suggestion
  class="first-class {% block extra_classes %}{% endblock %} modal fade"
```

Reply:
woops, absolutely. Just fixed it and updated the snapshots